app/plugin_api.go

```diff
@@ -279,7 +279,14 @@ func (api *PluginAPI) DeletePreferencesForUser(userId string, preferences []mode
 }
 
 func (api *PluginAPI) UpdateUser(user *model.User) (*model.User, *model.AppError) {
-	return api.app.UpdateUser(user, true)
+	updatedUser, err := api.app.UpdateUser(user, true)
+	if err != nil {
+		return nil, err
+	}
+
+	api.app.sendUpdatedUserEvent(*updatedUser)
```

Comment:
We should move this inside the `UpdateUser` method itself. There are 2 places where `sendUpdatedUserEvent` is called after UpdateUser. We should remove those as well.

Reply:
@agnivade This will send the UpdatedUser WebSocket event in other places a well like - 
`[UpdateUserNotifyProps](https://github.com/mattermost/mattermost-server/blob/c1dd23a3c80365c4ca4e2721e55c0c2c90038b25/app/user.go#L1348)` and `[ConvertBotToUser](https://github.com/mattermost/mattermost-server/blob/c1dd23a3c80365c4ca4e2721e55c0c2c90038b25/app/user.go#L2334)`

Would that be an acceptable change?

Reply:
Yes, I think that should be fine. Thanks.

Reply:
@agnivade done